python/ray/_private/log_monitor.py

```diff
@@ -67,6 +67,17 @@ def __init__(
         self.actor_name = None
         self.task_name = None
 
+    def reopen_if_necessary(self):
```

Comment:
Hmm are we doing the reopening strategy or limiting it to 1? We shouldn't have both.

Reply:
Turns out after fixing the typo you found, limiting to 1 doesn't work because we always reopen the same file, we don't queue/open the others.